Python/src/xpc.py

```diff
@@ -24,7 +26,9 @@ def __init__(self, xpHost = 'localhost', xpPort = 49009, port = 0, timeout = 100
             raise ValueError("Unable to resolve xpHost.")
 
         if xpPort < 0 or xpPort > 65535:
-            raise ValueError("The specified X-Plane port is not a valid port number.")
+            raise ValueError(
```

Comment:
Wrapping at 80 characters is just too aggressive for modern environments. No hard-wrapping at less than 120 characters please.